test/Exercism.Analyzers.CSharp.IntegrationTests/Exercism.Analyzers.CSharp.IntegrationTests.csproj

```diff
@@ -14,6 +14,9 @@
   <ItemGroup>
     <SolutionFiles Include="Solutions\**" />
     <Compile Remove="@(SolutionFiles)" />
+    <Content Include="@(SolutionFiles)">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
```

Comment:
I think this can be removed.

Reply:
If this is removed, Cs files are not in VS

Reply:
![image](https://user-images.githubusercontent.com/2979432/144018643-dabdf142-25a1-4d23-a27b-aa804df2b879.png)

Reply:
![image](https://user-images.githubusercontent.com/2979432/144018756-89ecdedf-55a1-4aa5-9e00-0fe807948fe2.png)

Reply:
Aha. Rider behaves differently then. Okay cool.